apps/web/src/features/swap/components/SwapOrder/rows/SurplusFee.tsx

```diff
@@ -7,13 +7,16 @@ import { HelpIconTooltip } from '@/features/swap/components/HelpIconTooltip'
 export const SurplusFee = ({
   order,
 }: {
-  order: Pick<Order, 'fullAppData' | 'sellToken' | 'buyToken' | 'status' | 'executedSurplusFee' | 'kind'>
+  order: Pick<Order, 'fullAppData' | 'sellToken' | 'buyToken' | 'status' | 'executedSurplusFee' | 'kind'> & {
+    // TODO: Remove when migrating to new SDK
+    executedFee: string
+    executedFeeToken: Order['sellToken']
+  }
```

Comment:
As we want to move to the newly generated types, would we prefer to update the old SDK instead?